src/main/java/com/r307/arbitrader/service/TradingScheduler.java

```diff
@@ -72,6 +73,10 @@ public TradingScheduler(
      */
     @PostConstruct
     public void connectExchanges() {
+
+        if (tradingConfiguration.getExitSpreadTarget() != null)
+            LOGGER.info("Property `exitSpreadTarget` is set, `minimumProfit` property setting will be ignored.");
```

Comment:
If you checked both `getExitSpreadTarget()` and `getMinimumProfit()` here then you could bump this up to a WARN. The warning is really if _both_ are set, so the user knows that their configuration is ambiguous and one setting will take precedence over the other. I don't need to see this message if I have only configured `exitSpreadTarget` because I probably understand what I've done.